src/main/java/com/exadel/discount/model/security/Role.java

```diff
@@ -0,0 +1,23 @@
+package com.exadel.discount.model.security;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@AllArgsConstructor
+@Getter
+public enum Role {
```

Comment:
I believe it should be just enum with 2 values (no need in separate permissions here)

Reply:
This is done.

Reply:
Move it to the entity layer. I believe User enetity will use it as well